### PR TITLE
Demonfish and Hisomukuro

### PR DESCRIPTION
**Pull request by xLightn1ng**
Fixed demonfish texture and created a swim animation
Added Hisomukuro

### CODE REVIEW

File: assets/minecraft/models/item/diamond_sword.json [hunk 1]
```diff
@@ -75,7 +75,7 @@
 		{ "predicate": {"custom_model_data": 69}, "model": "item/mob/chimokami/chimokami_run"},
 		{ "predicate": {"custom_model_data": 70}, "model": "item/mob/chimokami/chimokami_hit"},
 		{ "predicate": {"custom_model_data": 71}, "model": "item/mob/demonfish/demonfish"},
-		{ "predicate": {"custom_model_data": 72}, "model": "item/mob/demonfish/demonfish"},
```
Comment (xLightn1ng) [inline assets/minecraft/models/item/diamond_sword.json:78]: I saw that some of the mobs had two custom model data pathed to 1 model but others did not. I changed one of the extra paths to demonfish model and replaced it with a path to the demonfish_move model

Comment (0ffz) [reply assets/minecraft/models/item/diamond_sword.json:78]: Yep, anything that has a double for their second model is a placeholder for an animation that needs to be done

File: assets/minecraft/models/item/diamond_sword.json [hunk 2]
```diff
@@ -91,6 +91,8 @@
 		{ "predicate": {"custom_model_data": 85}, "model": "item/mob/titanjaw/titanjaw_hit"},
 		{ "predicate": {"custom_model_data": 86}, "model": "item/mob/stingerhead/stingerhead"},
 		{ "predicate": {"custom_model_data": 87}, "model": "item/mob/stingerhead/stingerhead"},
-		{ "predicate": {"custom_model_data": 88}, "model": "item/mob/stingerhead/stingerhead"}
+		{ "predicate": {"custom_model_data": 88}, "model": "item/mob/stingerhead/stingerhead"},
```
Comment (0ffz) [inline assets/minecraft/models/item/diamond_sword.json:94]: third should be stingerhead_hit

File: assets/minecraft/models/item/diamond_sword.json [hunk 3]
```diff
@@ -91,6 +91,8 @@
...
+		{ "predicate": {"custom_model_data": 89}, "model": "item/mob/hisomukuro/hisomukuro"},
+		{ "predicate": {"custom_model_data": 90}, "model": "item/mob/hisomukuro/hisomukuro_hit"}
```
Comment (0ffz) [inline assets/minecraft/models/item/diamond_sword.json:96]: the general pattern we use is
- idle
- moving
- hit

so please leave one number in between these two for the moving one, just set it to the same as the idle until moving animation comes around.

**Review by 0ffz — CHANGES_REQUESTED** [review]
:+1: for splitting up the textures when animating, i think having a huge texture is what causes the most lag :)

**Comment by Boy0000** [comment]
@0ffz can u remove the diamond_swird.json from this PR? Gonna push it through mine when that is ready, so since this is after that youknow. Also leave it open cuz lightning gonna keep pushing stuff to it

File: assets/minecraft/models/item/diamond_sword.json [hunk 4]
```diff
@@ -1,96 +0,0 @@
-{
```
Comment (0ffz) [inline assets/minecraft/models/item/diamond_sword.json:1]: This file shouldn't be deleted, revert it instead

**Review by 0ffz — CHANGES_REQUESTED** [review]
Everything seems great, just the diamond sword json that needs to be fixed

**Review by 0ffz — APPROVED** [review]

**Pull request merged**